build/allowedValues.json

```diff
@@ -10,7 +11,8 @@
     "7.6.2",
     "7.7.1",
     "7.8.1",
-    "7.9.0"
+    "7.9.4",
```

Comment:
[7.9.3 is the latest release in 7.9.x](https://www.elastic.co/downloads/past-releases/elasticsearch-7-9-3)

Reply:
amended to 7.9.3